src/ResourceManagement/OperationalInsights/Microsoft.Azure.Management.OperationalInsights/Generated/Models/Page2.cs

```diff
@@ -0,0 +1,42 @@
+// Code generated by Microsoft (R) AutoRest Code Generator 1.0.0.0
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+
+namespace Microsoft.Azure.Management.OperationalInsights.Models
+{
+
+    /// <summary>
+    /// Defines a page in Azure responses.
+    /// </summary>
+    /// <typeparam name="T">Type of the page content items</typeparam>
+    [Newtonsoft.Json.JsonObject]
+    public class Page2<T> : Microsoft.Rest.Azure.IPage<T>
+    {
+         /// <summary>
+         /// Gets the link to the next page.
+         /// </summary>
+         [Newtonsoft.Json.JsonProperty("@odata.nextLink")]
```

Comment:
These APIs are implemented long time ago. So the implementation may not match the ARM specification very well. I am just authroing the SDK in a way to match the implementation.